### PR TITLE
Updated README

### LINKED ISSUE #60: Add Test Suite content to README
Minimum add instructions on how to run the tests, although that could be in CONTRIBUTING.

### LINKED ISSUE #276: Documentation - provide a list of code bases showing exemplary use of python-cloudant
It would be great if we could provide a list of open source code bases that show python-cloudant being used as part of a bigger application such as flask and django.

I have put together a demo application with one sub-goal being to show python-cloudant used with flask:

 - https://github.com/snowch/movie-recommender-demo

The majority of the Cloudant code is in the following two files:

 - https://github.com/snowch/movie-recommender-demo/blob/master/web_app/app/dao.py 
 - https://github.com/snowch/movie-recommender-demo/blob/master/web_app/app/cloudant_db.py

The above app probably needs work before it can serve as an example.  I'm far from being an expert on python-cloudant!

### PR DESCRIPTION
**Pull request by emlaver**
## What

Updated README to markdown format and follow outline of other integration libraries.

## How

- Replaced README.rst with README.md
- Updated sub headers and sections to have the same outline as java-cloudant
- Added examples of this library in other open source projects

## Testing

No new tests.

## Issues
I believe we can close #60.  I've removed the Test Suite section and there's already testing environment instructions in CONTRIBUTING. 
#276

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Cloudant Python Client
+
+[![Build Status](https://travis-ci.org/cloudant/python-cloudant.svg?branch=master)](https://travis-ci.org/cloudant/java-cloudant)
```
Comment (ricellis) [inline README.md]: This is pointing to java-cloudant

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+[![Readthedocs](https://readthedocs.org/projects/pip/badge/)](http://python-cloudant.readthedocs.io)
+[![Compatibility](https://img.shields.io/badge/python-2.7%2C%203.5-blue.svg)](http://python-cloudant.readthedocs.io/en/latest/compatibility.html)
+[![pypi](https://img.shields.io/pypi/v/cloudant.svg)](https://pypi.python.org/pypi/cloudant)
+
+This is the official Cloudant library for Python.
+
+* [Installation and Usage](#installation-and-usage)
+* [Getting Started](#getting-started)
+* [API Reference](http://www.javadoc.io/doc/com.cloudant/cloudant-client/)
```
Comment (ricellis) [inline README.md]: Needs to point to the sub-section of readthedocs not javadoc.io

File: README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## Development
+
+See [CONTRIBUTING.rst](https://github.com/cloudant/python-cloudant/blob/master/CONTRIBUTING.rst)
+
+## Using in other projects
+
+The preferred approach for using `python-cloudant` in other projects is to use the PyPI as described above.
+
+#### Examples in open source projects
```
Comment (ricellis) [inline README.md]: Think this only needs 3 levels `###`

File: README.md (hunk 1 above)
Comment (emlaver) [reply README.md]: Fixed in 59d0747.

File: README.md (hunk 2 above)
Comment (emlaver) [reply README.md]: Fixed in 59d0747.

File: README.md (hunk 3 above)
Comment (emlaver) [reply README.md]: Fixed in 59d0747.

**Pull request merged**